packages/musap-rn-kms/src/agent/MusapKeyManagerSystem.ts

```diff
@@ -120,14 +121,16 @@ export class MusapKeyManagementSystem extends AbstractKeyManagementSystem {
     throw new Error('Not implemented.');
   }
 
-  private asMusapKeyInfo(args: MusapKey): ManagedKeyInfo & { keyUri?: string } {
-    return {
+  private asMusapKeyInfo(args: MusapKey): ManagedKeyInfo {
+    const keyInfo: Partial<ManagedKeyInfo> = {
       kid: args.keyId,
-      kms: args.sscdId,
       type: this.mapAlgorithmTypeToKeyType(args.keyType),
       publicKeyHex: args.publicKey.toString(),
-      keyUri: args.keyUri
```

Comment:
Why aren't we retuning the keyUri anymore?

Reply:
we're returning `keyUri` and every other value that MusapKey exposes in the meta property of IKey